rfc8446.xml

```diff
@@ -6067,6 +6075,7 @@ from older TLS implementations.</t>
        alert(21),
        handshake(22),
        application_data(23),
+       heartbeat(24),
```

Comment:
Can you add /* RFC 6520 */ after this line, as we did for the extension table above?